chapters/ch27.md

```diff
@@ -1,38 +1,37 @@
 # 27. Techniques for reducing variance
-->
+-> Các kỹ thuật làm giảm phương sai
 
 If your learning algorithm suffers from high variance, you might try the following techniques:
-->
+-> Nếu như thuật toán của bạn có phương sai lớn, bạn có thể thử các kỹ thuật sau: 
 
 
 * **Add more training data​**: This is the simplest and most reliable way to address variance, so long as you have access to significantly more data and enough computational power to process the data.
-->
+->* **Thêm dữ liệu huấn luyện**: Đây là cách đơn giản và đáng tin cậy nhất để giải quyết phương sai, miễn là bạn có quyền truy cập vào nhiều dữ liệu hơn và đủ sức mạnh tính toán để xử lý dữ liệu.
 
 
 * **Add regularization​** (L2 regularization, L1 regularization, dropout): This technique reduces variance but increases bias.
-->
+-> * ** Thêm regularization ** (L2 regularization, L1 regularization, dropout): Kỹ thuật này làm giảm phương sai nhưng tăng độ chệch.
 
 
 * **Add early stopping​** (i.e., stop gradient descent early, based on dev set error): This technique reduces variance but increases bias. Early stopping behaves a lot like regularization methods, and some authors call it a regularization technique.
-->
+-> * ** Thêm kỹ thuật dừng sớm ** (nghĩa là dừng kỹ thuật hạ dốc sớm, dựa trên lỗi của tập phát triển): Kỹ thuật này làm giảm phương sai nhưng tăng độ chệch. Kỹ thuật dừng sớm hành xử rất giống như các phương pháp regularization, và một số tác giả gọi đó là một kỹ thuật regularization.
```

Comment:
> Kỹ thuật dừng sớm hành xử  

nên chăng thay từ "hành xử" bằng "hoạt động"? "hành xử" nghe hơi con người, ko hợp cho thuật toán.

Reply:
Mình cũng nghĩ là từ "hoạt động" sẽ hay hơn. Nhưng ở đây mình có chút thắc mắc nếu để hoạt động có đúng ko? Early drop có tác dụng như regularization (force weights to have small values băngf việc để weights ko đi xa quá điểm init => phụ thuộc vào điểm init) nhưng nó có "hoạt động" giống như các regularization khác (ví dụ như L2 buộc weight to have small value) thì mình không chắc lắm. Ý bạn thấy thế nào?

Reply:
mình mượn tạm comment của @ngcthuong trong một discussion của bản dịch chap 25: "Regularization hiểu theo nghĩa là giới hạn không gian nghiệm lại theo một điều kiện nào đó"; theo nghĩa này thì cả dropout và regularization đều "hoạt động giống nhau", tức là giới hạn không gian nghiệm, cách giới hạn có thể khac nhau. Ở đây https://en.wikipedia.org/wiki/Dropout_(neural_networks) cũng có nói là "Dropout is a regularization technique patented by Google".